frontend/app/routes/logout.tsx

```diff
@@ -1,12 +1,34 @@
 import { sessionStorage } from "~/utilities/LoginHandler";
-import {
-  LoaderFunctionArgs,
-  redirect,
-} from "@remix-run/node";
+import { LoaderFunctionArgs, redirect } from "@remix-run/node";
+import { attemptLogout } from "~/utilities/fetchapi";
 
 export async function loader({ request }: LoaderFunctionArgs) {
-  let session = await sessionStorage.getSession(request.headers.get("cookie"));
+  const cookieHeader = request.headers.get("cookie");
+  let session = await sessionStorage.getSession(cookieHeader);
+  let user = session.get("user");
+
+  const cookies = Object.fromEntries(
+    cookieHeader
+      ?.split(";")
+      .map((cookie) => cookie.trim().split("=")) || []
+  );
+
+  attemptLogout(cookies["csrftoken"], cookies["sessionid"]);
+
+  let secure = "";
+  if (user?.backendCookie?.toLowerCase().includes("secure;"))
+    secure = " Secure;";
```

Comment:
```suggestion
let backendCookie: string[] = []
  if (user.backendCookie) {
    backendCookie = user.backendCookie
  }

  let headers = new Headers({ "set-cookie": await sessionStorage.destroySession(session) })

  backendCookie.forEach((cookie: string) => {
    let name = cookie.split("=", 1)[0]
    cookie = cookie.replace(/(sessionid|csrftoken)=[a-zA-Z0-9]*;/, `${name}=;`)
    cookie = cookie.replace(/[Ee]xpires=[a-zA-Z0-9 ,:]*;/, "Expires=Thu, 01 Jan 1970 00:00:00 GMT;");
    cookie = cookie.replace(/[Mm]ax-[Aa]ge=[0-9]*;/,"Max-Age=0;")
    headers.append("set-cookie", cookie);
  })
```
treat backendCookie as array, construct headers object with separate set-cookie headers and use String.replace with regex patterns to create the header for deleting a cookie